share/spice/seat_geek/sports/seat_geek_sports.js

```diff
@@ -0,0 +1,128 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_seat_geek_sports = function(api_result) {
+
+        if(!api_result || api_result.error || api_result.events.length === 0) {
+            return Spice.failed('seat_geek_sports');
+        }
+
+        var query = DDG.get_query();
+        var clean_query = query.replace(/((upcoming\s)?(match(es)?))|(events?)|(schedule)|(sports)/, '').trim().toLowerCase();
```

Comment:
Just FYI guys, we have a polyfill for `trim` in our JS code so it's actually OK to use it. Regardless, it feels weird to use something that technically isn't supported, so I'm all for using `$.trim`